todos.md

```diff
@@ -4,4 +4,4 @@
 - Given any schematic, Collect ALL resources and BUILD the schematic, ideally from a fresh survival world if possible.
 - Initialize and control multiple bots over a network. Kind of like an RTS game.
 - Make this bot able to survive anarchy
-- All acheivements, fully autonomous (not happening any time soon lol)
\ No newline at end of file
+- All achievements, fully autonomous (not happening any time soon lol)
```

Comment:
I approve of the merge, but I'll rope this PR in with a future update because I don't want to release a new change without other fixes, especially since the activity on the project is that of an archived one. Marvion is still an active maintainer and we will likely never implement all achievements mode.